src/common/pch.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+// Standard Library
+#include <chrono>
+#include <condition_variable>
+#include <cstdint>
+#include <fstream>
+#include <iostream>
+#include <map>
+#include <memory>
+#include <mutex>
+#include <queue>
+#include <string>
+#include <thread>
+#include <unordered_map>
+#include <vector>
+
+// Logging
+#define SPDLOG_ACTIVE_LEVEL SPDLOG_LEVEL_TRACE
```

Comment:
Might be able to remove this line. I think I added it a while ago when I couldn't get debug working, but from what I'm reading it shouldn't be needed. I'll have to check and test when I get back home.

Reply:
So actually, moving this define to a global header fixes the log behaviour, which I haven't realized was even broken! Thanks! However, now the logs are maybe a bit too verbose :grin:. I'll push a commit to fix that.

Reply:
Just realized that making `pch.h` not a precompiled header gives a weird warning about `SPDLOG_ACTIVE_LEVEL` being redefined. However, now that logging seems to be properly working, this warning probably isn't worth our time to look at.